src/server/conn/http1.rs

```diff
@@ -261,7 +261,7 @@ impl Builder {
     /// Note that this setting does not affect HTTP/2.
```

Comment:
There are a few comments with this note- can we remove these as well, since they no longer apply?

Reply:
Good catch, done.